modules/drivers/databricks/test/metabase/test/data/databricks.clj

```diff
@@ -163,11 +163,15 @@
                                            (rest sql-args))}
                           e)))))))
 
-;; With jdbc driver version 2.6.40 test data load fails due to ~statment using more parameters than driver's able to
-;; handle. `chunk-size` 25 works with 2.6.40, but dataset loading is really slow.
+;; 2.6.40 jdbc driver version statement param limit 256. Following implementation ensures test dataset loading won't
+;; exceed that. Orders table takes ~20 minutes to load.
```

Comment:
I think it would help to spell it out with an example. something like.

;; orders table has 12 columns in field def, id and a buffer is added
;; 256 / (12 + 2) = 18
;; so we can insert 18 rows at a time while staying under the param limit.